Cannot build project using flowbite 2.5.0

I am using flowbite in my project.  With v2.4.1, vite was able to build without issue.

Today, I updated to v2.5.0 and suddenly vite won't build it.  Here's what it's saying:

```
[vite:css] [postcss] Cannot find module 'typescript'
Require stack:
- /home/nticompass/Code/my_project/node_modules/flowbite/plugin.js
file: /home/nticompass/Code/my_project/resources/css/index.css:undefined:NaN
    at Module._resolveFilename (node:internal/modules/cjs/loader:1143:15)
    at Function.resolve (node:internal/modules/helpers:190:19)
    at _resolve (/home/nticompass/Code/my_project/node_modules/jiti/dist/jiti.js:1:241814)
    at jiti (/home/nticompass/Code/my_project/node_modules/jiti/dist/jiti.js:1:244531)
    at /home/nticompass/Code/my_project/node_modules/flowbite/plugin.js:6:23
    at evalModule (/home/nticompass/Code/my_project/node_modules/jiti/dist/jiti.js:1:247313)
    at jiti (/home/nticompass/Code/my_project/node_modules/jiti/dist/jiti.js:1:245241)
    at /home/nticompass/Code/my_project/tailwind.config.mjs:1:246
    at evalModule (/home/nticompass/Code/my_project/node_modules/jiti/dist/jiti.js:1:247313)
    at jiti (/home/nticompass/Code/my_project/node_modules/jiti/dist/jiti.js:1:245241)

Process finished with exit code 1
```

From what I can find, the issue seems to be the following line in `plugin.js`: https://github.com/themesberg/flowbite/blob/878ced1b622d1c48fece24bfbc4cd0064941bea6/plugin.js#L6

This import doesn't seem to be used anywhere in the file, and removing it seems to fix the issue.

--- a/plugin.js
+++ b/plugin.js
@@ -3,7 +3,6 @@ const svgToDataUri = require('mini-svg-data-uri');
 const plugin = require('tailwindcss/plugin');
 const defaultTheme = require('tailwindcss/defaultTheme');
 const colors = require('tailwindcss/colors');
-const { transform } = require('typescript');
 const [baseFontSize, { lineHeight: baseLineHeight }] =
     defaultTheme.fontSize.base;
 const { spacing, borderWidth, borderRadius, boxShadow } = defaultTheme;

